tests/agent/actions/test_generate_data_quality_check_expectations_action.py

```diff
@@ -76,12 +76,20 @@ def mock_data_asset(self, event: GenerateDataQualityCheckExpectationsEvent, asse
     def mock_metrics(self, data_asset: DataAsset[Any, Any]):
         return MetricRun(metrics=mock_metrics_list), uuid.uuid4()
 
+    def mock_no_anomaly_detection_coverage(self, data_asset: DataAsset[Any, Any]):
```

Comment:
Can you add some tests where we return some anomaly detection expectations and verify we don't create new ones?